docs/quickstart-vsphere.md

```diff
@@ -92,6 +92,7 @@ For the list of available settings along with their names please see the
 * `datastore_name` (optional, default = "datastore1") - vSphere datastore name
 * `network_name` (optional, default = "public") - vSphere network name
 * `template_name` (optional, default = "ubuntu-18.04") - vSphere template name to clone VMs from
+* `folder_name` (optional, default = "kubeone") - vSphere VM folder name
```

Comment:
`docs/` directory is deprecated and will be removed soon. Please send a PR to loodse/docs.

Reply:
Will do so. I suggest to keep it here as will until the doc gets removed